src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Orchestrators/Formatters/GivenACsvFormatter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.ProviderApprenticeshipsService.Web.Models.Agreement;
+using SFA.DAS.ProviderApprenticeshipsService.Web.Orchestrators.Formatters;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests.Orchestrators.Formatters
+{
+    [TestFixture]
+    public class GivenACsvFormatter
```

Comment:
I had thought we'd cover the ExcelFormatter in a unit test too? With ExcelFormatter as the sut, we'd pass a list of agreements to it just the same as here, but then we'd take the byte array output and run it back through the XLWorkbook constructor as a stream, asserting that the worksheet has got all the stuff we'd expect?